src/pages/settings/Security/CloseAccountPage.js

```diff
@@ -91,7 +93,7 @@ function CloseAccountPage(props) {
                 title={props.translate('closeAccountPage.closeAccount')}
                 onBackButtonPress={() => Navigation.goBack(ROUTES.SETTINGS_SECURITY)}
             />
-            <Form
```

Comment:
Seems Form can be also removed from import at line 19

Reply:
Of course, corrected it just now